src/app/basics/test-suites/overview/page.md

```diff
@@ -56,7 +56,7 @@ Because of this, a flaky test does not feel like an act of higher force, but can
 
 Replay Devtools work great with your existing test suite. Cypress and Playwright plugins allow you to **rewind or fast forward** to any command from your test run.
 
-The command details panel contains information on passed arguments, returned values and targeted element. Commands are integrated with the rest of DevTools. This means you can jump from a command detail into [Elements panel](/basics/replay-devtools/browser-devtools/elements-panel), or inspect API calls in the [Network panel](/basics/replay-devtools/browser-devtools/network-monitor).
+The command details panel contains information on passed arguments, returned values, and targeted elements. Commands are integrated with the rest of DevTools. This means you can jump from a command detail into the [Elements panel](/basics/replay-devtools/browser-devtools/elements-panel), or inspect API calls in the [Network panel](/basics/replay-devtools/browser-devtools/network-monitor).
```

Comment:
I'm partial to the Oxford comma. Regardless, I think we should choose one and make that the law of the land.

Reply:
https://www.oxford-royale.com/articles/oxford-comma/#:~:text=When%20you're%20writing%20a,marshmallows%2C%20cake%20and%20chicken.%E2%80%9D

Why I like the Oxford comma.

Reply:
> I'm partial to the Oxford comma.

Same

> I think we should choose one and make that the law of the land.

Big same

Reply:
I feel very strongly about this as well.

1) Yes Oxford comma (and honestly the counter-arguments are not compelling at all)
2) Yes to consistency